beacon_node/beacon_chain/src/metrics.rs

```diff
@@ -1900,13 +1852,6 @@ pub static DATA_AVAILABILITY_OVERFLOW_MEMORY_STATE_CACHE_SIZE: LazyLock<Result<I
             "Number of entries in the data availability overflow state memory cache.",
         )
     });
-pub static DATA_AVAILABILITY_OVERFLOW_STORE_CACHE_SIZE: LazyLock<Result<IntGauge>> =
-    LazyLock::new(|| {
-        try_create_int_gauge(
-            "data_availability_overflow_store_cache_size",
```

Comment:
We have `DATA_AVAILABILITY_OVERFLOW_MEMORY_BLOCK_CACHE_SIZE` and `DATA_AVAILABILITY_OVERFLOW_MEMORY_STATE_CACHE_SIZE`, so ok to remove